orchestration/dags/dependencies/firebase/sql/analytics/aggregated_daily_offer_consultation_data.sql

```diff
@@ -0,0 +1,62 @@
+SELECT
+    fe.event_date,
+    o.offerer_id,
+    o.offerer_name,
+    o.venue_id,
+    o.venue_name,
+    o.offer_id,
+    o.offer_name,
+    c.name,
+    fe.event_name,
+    fe.traffic_medium,
+    fe.traffic_campaign,
+    fe.origin,
+    CASE
+        WHEN fe.user_id IS NOT NULL
+        AND DATETIME(fe.event_timestamp) BETWEEN eud.user_deposit_creation_date
+        AND eud.user_deposit_expiration_date THEN 'Bénéficiaire'
+        WHEN fe.user_id IS NOT NULL
+        AND DATETIME(fe.event_timestamp) > eud.user_deposit_expiration_date THEN 'Ancien bénéficiaire'
```

Comment:
A voir comment on veut déterminer ce qu'est un ancien bénéficiaire : quid de ceux dont le crédit est épuisé (montant restant = 0) ?

Reply:
C'est noté, @adrienlin travaille sur une table de profil d'utilisateur afin que cette règle puisse être appliquée dans d'autres cas & TdB. On peut prendre cela en compte lors de la prochaine itération.